utils/Cargo.toml

```diff
@@ -12,3 +12,4 @@ miden_objects = { workspace = true }
 serde = { version = "1.0", features = ["derive"] }
 tracing = { workspace = true }
 tracing-subscriber = { workspace = true }
+miden-node-utils-macro = { path = "../utils-macro" }
```

Comment:
This is needed only for testing, right? If so, it would be better to import it as a `dev-dependency` in crates where it is used.

Reply:
This crate is re-exporting the symbol, so that is not a dev only dependency. Gated behind a `testing` feature flag.